apps/frontend/src/libs/components/activity-chart/activity-chart.tsx

```diff
@@ -0,0 +1,22 @@
+import { Line, LineChart } from "recharts";
+
+import { type ActivityChartData } from "./libs/types/types.js";
+
+type Properties = {
+	data: ActivityChartData;
+};
+
+const ActivityChart = ({ data }: Properties): JSX.Element => {
+	return (
+		<LineChart data={data} height={40} width={120}>
+			<Line
+				dataKey="commitsNumber"
```

Comment:
I think we shouldn't be attached to commitsNumber inside this component, we should be able to reuse it not only for commits

Reply:
Okay. Should I assume that we should be able to use it not only for activity charts, and rename it to `Chart` as well?

Reply:
yes, this will be better I think

Reply:
Done